File: c++/08_pgm_invoice_engine/src/co/edu/campusucc/poo/app/MainInvoiceEngine.cpp
#include <iostream>
#include "Client.h"
#include "Products.h"
#include "Invoice.h"

using namespace std;

int main() {
    Client clie;
    Products prod;
    Invoice invo;

    clie.nombre();
    clie.apellido();
    clie.id();
    prod.nombreProducto();
    prod.id();
    prod.precio();
    invo.tipo();
    invo.id();
    clie.tipoTarjeta();
    clie.numTarjeta();
    invo.total();

    return 0;
}

File: ruby/pgms_ruby/08-pgmInvoiceEngine/MainInvoiceEngine.rb
require_relative 'Client'
require_relative 'Invoice'
require_relative 'Products'

def main
    clie = Client.new
    prod = Products.new
    invo = Invoice.new

    clie.nombre
    clie.apellido
    clie.id
    prod.nombreProducto
    prod.id
    prod.precio
    invo.tipo
    invo.id
    clie.tipoTarjeta
    clie.numTarjeta
    invo.total
end

if __FILE__ == $0
    main
end